$NetBSD: patch-pyproject.toml,v 1.1 2025/04/21 14:11:02 schmonz Exp $

Apply upstream patches:

183a17a for lighter TOOL_DEPENDS
eda99a9 to fix a deprecation warning

--- pyproject.toml.orig	2022-08-26 14:54:21.000000000 +0000
+++ pyproject.toml
@@ -10,12 +10,12 @@ python = "^3.6"
 feedparser = ">=6.0.5"
 html2text = ">=2020.1.16"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 update-copyright = ">=0.6.2"
 
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
 r2e = "rss2email.main:run"
